diff --git a/node_modules/react-datocms/dist/types/StructuredText/index.d.ts b/node_modules/react-datocms/dist/types/StructuredText/index.d.ts
index 5e55966..1ea35fd 100644
--- a/node_modules/react-datocms/dist/types/StructuredText/index.d.ts
+++ b/node_modules/react-datocms/dist/types/StructuredText/index.d.ts
@@ -25,19 +25,19 @@ declare type RenderRecordLinkContext<R extends StructuredTextGraphQlResponseReco
 declare type RenderBlockContext<R extends StructuredTextGraphQlResponseRecord> = {
     record: R;
 };
-export declare type StructuredTextPropTypes<R extends StructuredTextGraphQlResponseRecord> = {
+export declare type StructuredTextPropTypes<R1 extends StructuredTextGraphQlResponseRecord, R2 extends StructuredTextGraphQlResponseRecord> = {
     /** The actual field value you get from DatoCMS **/
-    data: StructuredTextGraphQlResponse<R> | StructuredTextDocument | Node | null | undefined;
+    data: StructuredTextGraphQlResponse<R1, R2> | StructuredTextDocument | Node | null | undefined;
     /** A set of additional rules to convert nodes to JSX **/
     customNodeRules?: RenderRule<H, T, F>[];
     /** A set of additional rules to convert marks to JSX **/
     customMarkRules?: RenderMarkRule<H, T, F>[];
     /** Fuction that converts an 'inlineItem' node into React **/
-    renderInlineRecord?: (context: RenderInlineRecordContext<R>) => ReactElement | null;
+    renderInlineRecord?: (context: RenderInlineRecordContext<R2>) => ReactElement | null;
     /** Fuction that converts an 'itemLink' node into React **/
-    renderLinkToRecord?: (context: RenderRecordLinkContext<R>) => ReactElement | null;
+    renderLinkToRecord?: (context: RenderRecordLinkContext<R2>) => ReactElement | null;
     /** Fuction that converts a 'block' node into React **/
-    renderBlock?: (context: RenderBlockContext<R>) => ReactElement | null;
+    renderBlock?: (context: RenderBlockContext<R1>) => ReactElement | null;
     /** Function that converts 'link' and 'itemLink' `meta` into HTML props */
     metaTransformer?: TransformMetaFn;
     /** Fuction that converts a simple string text into React **/
@@ -49,4 +49,4 @@ export declare type StructuredTextPropTypes<R extends StructuredTextGraphQlRespo
     /** @deprecated use customNodeRules **/
     customRules?: RenderRule<H, T, F>[];
 };
-export declare function StructuredText<R extends StructuredTextGraphQlResponseRecord>({ data, renderInlineRecord, renderLinkToRecord, renderBlock, renderText, renderNode, renderFragment, customMarkRules, customRules, customNodeRules, metaTransformer, }: StructuredTextPropTypes<R>): ReactElement | null;
+export declare function StructuredText<R1 extends StructuredTextGraphQlResponseRecord, R2 extends StructuredTextGraphQlResponseRecord>({ data, renderInlineRecord, renderLinkToRecord, renderBlock, renderText, renderNode, renderFragment, customMarkRules, customRules, customNodeRules, metaTransformer, }: StructuredTextPropTypes<R1, R2>): ReactElement | null;
